modules/ROOT/pages/workday/workday-connector-studio.adoc

```diff
@@ -32,35 +31,35 @@ To configure a connector in Anypoint Studio:
 
 == Configure an Input Source
 
-Configure an input source for the connector such as an operation for
-the Workday connector, *HTTP Listener*, or *Scheduler*.
-
-You can put the following operation for the connector in
-the *Source* area of the Studio canvas:
+Configure an input source for the connector.  Possible input source operations are:
 
-New objects trigger
+* *On New Objects* - Start your app when an object is created in Workday. 
+* *HTTP Listener* - Start your app using a browser or a command such as `curl`.
+* *Scheduler* - Start your app at timed intervals.
```

Comment:
GLOBALLY reformat lists like these to the following format:

* *On New Objects*
    Starts your app when an object is created in Workday

Please review style reference.